dist/mapbox-gl.css

```diff
@@ -303,6 +343,65 @@ a.mapboxgl-ctrl-logo {
     will-change: transform;
 }
 
+.mapboxgl-user-location-dot {
+    background-color: #1DA1F2;
+    width: 20px;
```

Comment:
It feels a tad tiny at 12px, to the point where it almost dissapears in the map at a glance, and compared to the size of other UI elements (like zoom buttons).

Noting at https://github.com/mapbox/mapbox-gl-js/pull/3781#discussion_r98823742 that the design came from @tristen .